### PR TITLE
refactor: remove unused commands

### PR DESCRIPTION
**Pull request by fmartingr**
#### Summary
- Removed commands not present in documentation: Create calendar, Delete calendar, Find Meetings, Create Event
- Moved the show calendars command to admin only, in preparation for when we start working on multiple calendar support.

### CODE REVIEW

File: calendar/command/command.go [hunk 1]
```diff
@@ -117,21 +117,13 @@ func (c *Command) Handle() (string, bool, error) {
...
-	case "showcals":
-		handler = c.requireConnectedUser(c.showCalendars)
 	case "settings":
 		handler = c.requireConnectedUser(c.settings)
 	case "events":
 		handler = c.requireConnectedUser(c.event)
-	// Admin only
+		// Admin only
+	case "showcals":
+		handler = c.requireConnectedUser(c.requireAdminUser(c.showCalendars))
```
Comment (mickmister) [inline calendar/command/command.go:126]: nit on formatting:

```suggestion
	// Admin only
	case "showcals":
		handler = c.requireConnectedUser(c.requireAdminUser(c.showCalendars))
```

**Review by mickmister — APPROVED** [review]
LGTM, thanks for this cleanup :+1:

**Comment by raghavaggarwal2308** [comment]
> Added the suggestion and fixed the CI. Blocked until #267 is merged.

@fmartingr #267 is merged, can we merge this PR?

**Comment by fmartingr** [comment]
> > Added the suggestion and fixed the CI. Blocked until #267 is merged.
> 
> @fmartingr #267 is merged, can we merge this PR?

Sure, go ahead.

**Review by wiggin77 — APPROVED** [review]

**Comment by raghavaggarwal2308** [comment]
@wiggin77 @fmartingr Do you think we need a QA on this?

**Comment by wiggin77** [comment]
> @wiggin77 @fmartingr Do you think we need a QA on this?

I don't think so, but Felipe can decide as he is the author.

**Comment by fmartingr** [comment]
> @wiggin77 @fmartingr Do you think we need a QA on this?

I don't think we need QA on this, I just removed a bunch of unsupported /non working commands.

**Pull request merged**